gocardless_pro/resources/billing_request.py

```diff
@@ -56,6 +56,11 @@ def payment_request(self):
         return self.PaymentRequest(self.attributes.get('payment_request'))
   
 
+    @property
+    def purpose_code(self):
+        return self.attributes.get('purpose_code')
```

Comment:
I assume this is due to previous changes that were not released?

Reply:
Yes, I think this is a result of me updating the order of the billing requests in the schema so that the docs follow logically. https://github.com/gocardless/payments-service/pull/40145

Reply:
Yeah there was a similar one for dotnet when i released earlier https://github.com/gocardless/gocardless-dotnet-template/pull/163